server/schedulers/hot_region_config.go

```diff
@@ -311,7 +311,7 @@ func (conf *hotRegionSchedulerConfig) handleSetConfig(w http.ResponseWriter, r *
 	newc, _ := json.Marshal(conf)
 	if !bytes.Equal(oldc, newc) {
 		conf.persist()
-		rd.Text(w, http.StatusOK, "success")
+		rd.Text(w, http.StatusOK, "success ")
```

Comment:
```suggestion
		rd.Text(w, http.StatusOK, "success")
```

Reply:
change the hot region config ouput will return "successno change", I think this result format is not clear